src/tests/test_api.py

```diff
@@ -59,25 +62,36 @@
     "/origin/grid-mapfile",
 ]
 
+JSON_ENDPOINTS = [
+    "/api/resource_group_summary",
+    '/miscproject/json',
+    '/miscsite/json',
+    '/miscfacility/json',
+    '/miscresource/json',
+    '/vosummary/json',
+    '/resources/stashcache-files',
+    '/stashcache/namespaces'
+]
+
 
 @pytest.fixture
-def client():
+def client() -> FlaskClient:
```

Comment:
```suggestion
def client() -> Generator[FlaskClient]:
```
since you're using a `yield` statement